netbox_topology_views/views.py

```diff
@@ -200,14 +201,29 @@ def create_node(
         if not group_id:
             print('Exception occured while handling default group.')
             return node
-   
+
     group = get_object_or_404(CoordinateGroup, pk=group_id)
 
-    node["physics"] = True
-    # Coords must be set even if no coords have been stored. Otherwise nodes with coords 
-    # will not be placed correctly by vis-network.
-    node["x"] = 0
-    node["y"] = 0
+    node["physics"] = not disable_physics
+    if not disable_physics:
+        # Coords must be set even if no coords have been stored. Otherwise nodes with coords
+        # will not be placed correctly by vis-network.
+        node["x"] = 0
+        node["y"] = 0
+    else:
+        # draw devices clustered by their rack neighbours
+        # for center coordinates of a rack, use the rack name as seed (if available)
+        random.seed(device.rack.name if hasattr(device, "rack") else device.name)
```

Comment:
```suggestion
        random.seed(device.rack.name if hasattr(device, "rack") else device.site)
```
`site` is a mandatory field when creating a new Device. Also would remove the `if` in line 222, simply setting the new seed to `device-name` for all Devices.